src/node.cc

```diff
@@ -5490,8 +5490,8 @@ void StopEventLoop() {
   request_stop = true;
 }
 
-bool ProcessEvents() {
```

Comment:
is this function effectively the same as TickEventLoop now? do we even still need it?

Reply:
Talked about it in person. `TickEventLoop()` is an internal function which is why `ProcessEvents()` is required.